stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -912,14 +915,19 @@
           (asserts! (is-eq pox-addr (get pox-addr existing-entry))
                     (err ERR_DELEGATION_WRONG_REWARD_SLOT))
 
+          ;; Validate that amount is less than or equal to `max-amount`
+          (asserts! (>= max-amount increased-ustx) (err ERR_SIGNER_AUTH_AMOUNT_TOO_HIGH))
+
+          ;; Verify signature from delegate that allows this sender for this cycle
+          (try! (consume-signer-key-authorization pox-addr reward-cycle "agg-increase" u1 signer-sig signer-key increased-ustx max-amount auth-id))
+
           ;; update the pox-address list -- bump the total-ustx
           (map-set reward-cycle-pox-address-list
                    { reward-cycle: reward-cycle, index: reward-cycle-index }
                    { pox-addr: pox-addr,
                      total-ustx: increased-ustx,
```

Comment:
Not a problem with your change, but this naming scares me and wanted to point it out here. It would be very easy to make the same mistake from pox-2 here because of `total-ustx` and `increased-ustx`.

Reply:
Changed variables accordingly

Reply:
Much better, and clearly those names were a problem! Thanks!